easybuild/easyconfigs/g/gmpy2/gmpy2-2.1.0b1-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'gmpy2'
+version = '2.1.0b1'
```

Comment:
@smoors Why `2.1.0b1` if there's also a `2.1.0b4` (for a while already)?

Reply:
@boegel because `2.1.0b4` is not advertized as a release, but no problem for me if you prefer it.